apps/participants/urls.py

```diff
@@ -38,4 +38,9 @@
         views.participant_team_detail,
         name="get_participant_team_details",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/team_detail/$",
```

Comment:
Let's change the url pattern to `r"^challenges/(?P<challenge_pk>[0-9]+)/team_details/$"`